cli/src/main/kotlin/com/meamoria/lexurgy/cli/server/Server.kt

```diff
@@ -1,21 +1,51 @@
 package com.meamoria.lexurgy.cli.server
 
 import com.meamoria.lexurgy.cli.soundChangerFromLscFile
+import com.meamoria.lexurgy.cli.soundChangerFromString
+import com.meamoria.lexurgy.sc.SoundChanger
 import kotlinx.serialization.SerialName
 import kotlinx.serialization.Serializable
 import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.JsonElement
+import kotlinx.serialization.json.jsonObject
+import kotlinx.serialization.json.jsonPrimitive
 import java.nio.file.Path
+import java.nio.file.Paths
 
+enum class RequestType {
+    // This is not exactly according to Kotlin conventions, but it's the
+    // simplest way to get it to serialize the enumerators exactly as we
+    // want without having to add a lot of boilerplate.
+    apply,
+    load_file,
+    load_string
+}
 
 @Serializable
 data class ServerRequest(
```

Comment:
Should be replaced by a sealed class hierarchy. See [https://github.com/Kotlin/kotlinx.serialization/blob/master/docs/polymorphism.md]